WordPress/src/main/java/org/wordpress/android/models/Person.java

```diff
@@ -1,33 +1,62 @@
 package org.wordpress.android.models;
 
+import org.json.JSONObject;
+
 public class Person {
-    public long personID;
+    private long personID;
+    private int localTableBlogId;
 
     private String username;
     private String firstName;
     private String lastName;
     private String displayName;
-    private String imageUrl;
+    private String avatarUrl;
     private Role role;
 
     public Person(long personID,
+                  int localTableBlogId,
                   String username,
                   String firstName,
                   String lastName,
                   String displayName,
-                  String imageUrl,
+                  String avatarUrl,
                   Role role) {
         this.personID = personID;
+        this.localTableBlogId = localTableBlogId;
         this.username = username;
         this.firstName = firstName;
         this.lastName = lastName;
         this.displayName = displayName;
-        this.imageUrl = imageUrl;
+        this.avatarUrl = avatarUrl;
         this.role = role;
     }
 
+    public static Person fromJSON(JSONObject json, int localTableBlogId) {
+        if (json == null) {
+            return null;
+        }
+
+        long personID = Long.parseLong(json.optString("ID"));
+        String username = json.optString("login");
+        String firstName = json.optString("first_name");
+        String lastName = json.optString("last_name");
+        String displayName = json.optString("nice_name");
+        String avatarUrl = json.optString("avatar_URL");
+        Role role = Role.fromKey(json.optJSONArray("roles").optString(0));
```

Comment:
Shall we add a minimal comment here? Something along the lines "we don't support multiple roles yet so, just pick up the first role in the list" or perhaps something more accurate you might have in mind Oguz. Cheers!